tests/raw_query/tests.py

```diff
@@ -180,6 +180,12 @@ def test_params(self):
         self.assertEqual(len(results), 1)
         self.assertIsInstance(repr(qset), str)
 
+    def test_like_param(self):
+        query = "SELECT * FROM raw_query_author WHERE first_name like 'J%'"
+        qset = Author.objects.raw(query, params=None)
+        results = list(qset)
+        self.assertEqual(len(results), 2)
```

Comment:
Can we add an extra tests (in a separate commit) without passing params? For example:
```python
query = "SELECT * FROM raw_query_author WHERE first_name like 'J%%'"
qset = Author.objects.raw(query)
```